idep/assetlist.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "idep",
+  "assets": [
+    {
+      "description": "The native token of IDEP chain",
+      "denom_units": [
+        {
```

Comment:
Can you please also add a 0-exponent denom? Based on what I see on the explorer, it looks like the base unit is 'idep' and the practical token is 'IDEP' (or 1 million 'idep'). Then we can capitalize 'idep' to 'IDEP' for the 6-exponent denom.

Reply:
oof - let's not take the block explorer UI as gospel. `uidep` would be much easier to understand

Reply:
Done @JeremyParish69

Reply:
If you have any changes to request @blake-regalia, please contact the IDEP team directly, here it is to validate that the PR is correct, which is the case according to the network.